src/app/pages/lists/projects/listItemView.tsx

```diff
@@ -19,6 +19,7 @@ function ListItemView(props) {
                 </Item.Meta>
                 <Item.Description>
                     <ReactMarkdown source={props.data.content} />
+                    {props.data.technologies.map(item => <div className="ui basic label">{item}</div>)}
```

Comment:
`Item.Description` içerisine bunu almak istemeyiz, `Item.Extra` daha doğru bir alan olabilri.

ayrıca **Semantic UI React** kullandığımız için className'i elle vermek yerine UI Kit'in kendi componentlerini kullanmak daha anlamlı olacak.

Reply:
Yine label'ları bir "label group" içerisine almak daha iyi olabilir.

Reply:
Label kullanıldı, label group ve item.extra yapıldı.